pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -46,33 +45,31 @@
     private transient ServiceUrlProvider serviceUrlProvider;
 
     @JsonIgnore
-    private transient Authentication authentication = new AuthenticationDisabled();
-    @JsonIgnore
-    private transient String authPluginClassName;
-    @JsonIgnore
-    private transient Map<String, String> authParams;
+    @Builder.Default private transient Authentication authentication = new AuthenticationDisabled();
+    private String authPluginClassName;
+    private String authParams;
 
-    private long operationTimeoutMs = 30000;
-    private long statsIntervalSeconds = 60;
+    @Builder.Default private long operationTimeoutMs = 30000;
```

Comment:
All these annotations seem to go a bit against the spirit of Lombok :) which is there to reduce the boiler-plate code.

Could we instead remove the `@Builder` and rely on the declared defaults (and just use `setX()` ) ?

Reply:
Builder is just a convenience layer so of course yes. Requires changes in tests that use builder so I will come back with all the changes in  next 3-4 days @merlimat :-)